manual/guides/blue_waters.md

```diff
@@ -30,24 +31,19 @@ follow the instruction sent to you by email.
 Wait patiently for the token to arrive.
 
 ## Step 4. Activating Token
-With your token, open the email 'Blue Waters Instructions'
-Click the [link](https://otp.ncsa.illinois.edu)
-and follow its directions, and activate your token.
+For activating the token, instructions should be contained within the emails recieved. Extra information can be found [here](https://wiki.ncsa.illinois.edu/display/cybersec/Duo+at+NCSA).
 
-** tip) the pin must be 8 digits, letters and numbers, something that you'll never forget!
 
 
 ## Step 5. Loggin In
 Open terminal, type
 ```
 $ssh [username]@h2ologin.ncsa.illinois.edu
 ```
+You will then be prompted for your username, password, token selection, and then your token input.
```

Comment:
I'm confused by this sentence. When SSH-ing in, you've already given your username (in the ssh command above) and there's no need to select a token (that was done in steps 2,3, & 4.) At this point, you just type in your pin along with whatever number appears on your token. What am I missing?

Reply:
Ah, the username part was a mistake. When I SSH in though, it has a list (though there is only one option) where it says "Enter a passcode or select one of the following options" followed by "1. Duo Push to XXX-XXX-0603". After I type "1" to select that option, I can accept the request from Duo on my phone and gain access. I think this is some sort of token selection list, where if someone has a a physical token and Duo they could use either.

Reply:
ah, I see. I forgot about that. Maybe clarify all that here.